go.mod

```diff
@@ -0,0 +1,51 @@
+module github.com/volcano-sh/kubesim
+
+go 1.15
+
+require (
+	github.com/davecgh/go-spew v1.1.2-0.20180830191138-d8f796af33cc // indirect
+	github.com/go-sql-driver/mysql v1.5.0
+	github.com/golang/mock v1.3.1
+	github.com/google/cadvisor v0.35.0
+	github.com/pmezard/go-difflib v1.0.1-0.20181226105442-5d4384ee4fb2 // indirect
+	github.com/spf13/cobra v0.0.5
+	github.com/spf13/pflag v1.0.5
+	github.com/stretchr/objx v0.3.0 // indirect
+	github.com/stretchr/testify v1.6.2-0.20200628121210-87a988cffbb9
+	google.golang.org/grpc v1.26.0
+	gopkg.in/yaml.v3 v3.0.0-20200615113413-eeeca48fe776 // indirect
+	k8s.io/api v0.0.0
+	k8s.io/apimachinery v0.0.0
+	k8s.io/client-go v0.0.0
+	k8s.io/component-base v0.0.0
+	k8s.io/cri-api v0.0.0
+	k8s.io/klog v1.0.0
+	k8s.io/kubernetes v0.0.0-20200630205207-e38139724f8f
```

Comment:
let's add version to k8s related repo :)

Reply:
> let's add version to k8s related repo :)

there is no tag version for commit e38139724f8f0086be9db575d8fe5211c653dc8f, that pointed in README.md
nearest tagged commit is dff82dc0de47299ab66c83c626e08b245ab19037 v1.18.6
should am i use that?

Reply:
`v1.18.6` is ok to me :) please also update other related repo versions :)